.github/workflows/reports.yml

```diff
@@ -0,0 +1,29 @@
+name: CI Reports
+
+on:
+  workflow_call:
+  schedule:
+    - cron: "0 0 * * 1" # At 0:00 on Monday
```

Comment:
How long does it take to run? isn't it better to run it at 7 am instead?

Reply:
The local runs were under 4 minutes for both. I guess 10 minutes tops on the CI?

Why is 7 AM better? we want the data from last week, which ends at Monday midnight

Reply:
While the message would not tag anyone, sending a message to our channel in the middle of the night from Monday to Sunday doesn't sound a good idea.